packages/app-admin-ui/server/env.js

```diff
@@ -1,4 +1,4 @@
 exports.mode = process.env.NODE_ENV === 'production' ? 'production' : 'development';
 
 exports.enableDevFeatures =
-  process.env.ENABLE_DEV_FEATURES === 'true' ? true : !(exports.mode === 'production');
+  process.env.ENABLE_DEV_FEATURES === 'true' ? true : exports.mode !== 'production';
```

Comment:
DeepScan complained about this for some reason... Did their config change? If so, how do we stop that from blocking PRs in the future?

https://deepscan.io/dashboard/#view=project&tid=3252&pid=4799&bid=38273&subview=pull-request&prid=114374

Reply:
A while ago, I was talking to @timleslie and we thought we should remove DeepScan because there are so many false positives and we aren't really getting any value out of it now.